odoo/addons/base/models/assetsbundle.py

```diff
@@ -173,8 +173,17 @@ def to_node(self, css=True, js=True, debug=False, async_load=False, defer_load=F
                 ])
                 response.append(("link", attr, None))
             if self.css_errors:
-                msg = '\n'.join(self.css_errors)
-                response.append(JavascriptAsset(self, inline=self.dialog_message(msg)).to_node())
+                msg = '\n'.join(self.css_errors).replace('"', '\\"').replace('\n', '\\n')
+                js_error = """
+                    window.__odooScssCompilationError = "%s";
+                    console.error("SCSS compilation failure:", window.__odooScssCompilationError);
+                    setTimeout(() => {
+                        if (!odoo || !odoo.define) {
+                            alert(window.__odooScssCompilationError);
+                        }
+                    }, 500);
```

Comment:
why do we need a timeout of half a second ?

Reply:
yes I think we should look in that direction, because waiting a random amount of time will `sometimes` not work.